--- SguildTranscript.tcl	2011-06-08 08:43:39.000000000 -0300
+++ SguildTranscript.tcl.n	2012-08-16 17:46:38.000000000 -0300
@@ -381,3 +381,21 @@
     }
 
 }
+
+proc CliScript { clientSocketID eventInfo } {
+
+    if { [llength $eventInfo] == 7 } {
+
+        lassign $eventInfo \
+            sensor timestamp sensorID srcIP dstIP srcPort dstPort
+
+        XscriptRequest $clientSocketID $sensor $sensorID CLI $timestamp $srcIP $srcPort $dstIP $dstPort 0
+
+    } else {
+
+        SendSocket $clientSocketID [list XscriptMainMsg CLI "Request Failed"]
+        SendSocket $clientSocketID [list XscriptMainMsg CLI DONE]
+
+    }
+
+}
